src/ajax/core.cljs

```diff
@@ -15,6 +15,8 @@
   (:require-macros [ajax.macros :as m]
                    [poppea :as p]))
 
+(def default-interceptors (atom ()))
```

Comment:
Should we be storing the interceptor map as `[{:req fn :res fn}]` or as `{:req [fn] :res [fn]}`? I think the latter is faster at runtime.